ceph-website-prs/build/build

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ex
+
+## TODO: Check if only the src/ dir changed and `exit 0` if only the ceph.io.git README or something else inconsequential changed.
+env
+
+BRANCH=$(echo $GIT_BRANCH | sed 's:.*/::')
+
+npm install
```

Comment:
Hey @djgalloway — one thing crossed my mind looking at this PR. We might benefit from replacing our `npm install` with `npm ci` ([docs](https://docs.npmjs.com/cli/v7/commands/npm-ci)) in CI environments. Expect, if nothing else, it might speed up builds 👍